Lib/multiprocessing/pool.py

```diff
@@ -151,8 +151,9 @@ class Pool(object):
     '''
     _wrap_exception = True
 
-    def Process(self, *args, **kwds):
-        return self._ctx.Process(*args, **kwds)
+    @staticmethod
+    def Process(ctx, *args, **kwds):
+        return ctx.Process(*args, **kwds)
```

Comment:
Why did this need to change? I feel like `pool.Process` existed solely as a convenience method, and this breaks any code that used it. Instead of changing this function, you could just use `ctx.Process` instead of `self.Process` below